components/screens/menu/Version.ts

```diff
@@ -0,0 +1,15 @@
+import * as Application from 'expo-application';
+import * as Updates from 'expo-updates';
+import {getUpdateTimeAsVersionString} from 'hooks/useEASUpdateStatus';
+import {Platform} from 'react-native';
+
+export const getVersionInfoFull = (mixpanelUserId: string, updateGroupId: string): string => {
+  const versionTestFull = `Avy version ${Application.nativeApplicationVersion || 'n/a'} (${Application.nativeBuildVersion || 'n/a'})
+Build date ${getUpdateTimeAsVersionString()}
+Git revision ${process.env.EXPO_PUBLIC_GIT_REVISION || 'n/a'}
+Update group ID ${updateGroupId || 'n/a'} (channel: ${Updates.channel || 'development'})
+Update ID ${Updates.updateId || 'n/a'} (platform: ${Platform.OS})
+User ID ${mixpanelUserId || 'n/a'}`;
```

Comment:
Ah, looks like you are already doing the `|| 'n/a'` - so you should be able to just update the function signature.